docs/deployment/advanced/platform.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Customizable Features for Platform Users"
+nav-title: platform
+nav-parent_id: advanced
+nav-pos: 3
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This documentation page is hard to read in my opinion. It should first describe on a high level that a user can register multiple exception listeners, which are called each time an exception is reported at runtime.
The purpose of these listeners is to build metrics based on the exceptions, make call to external systems or classify the exceptions otherwise. ...

Reply:
Revised the intro part as your suggested.